content/en/blog/_posts/XXXXXX-kubernetes-v1-35-sneak-peek.md

```diff
@@ -0,0 +1,149 @@
+
+- Generally available (GA) or stable API versions may be marked as deprecated but must not be removed within a major version of Kubernetes.
+- Beta or pre-release API versions must be supported for 3 releases after the deprecation.
+- Alpha or experimental API versions may be removed in any release without prior deprecation notice; this process can become a withdrawal in cases where a different implementation for the same feature is already in place.
+
+Whether an API is removed as a result of a feature graduating from beta to stable, or because that API simply did not succeed, all removals comply with this deprecation policy. Whenever an API is removed, migration options are communicated in the [deprecation guide](/docs/reference/using-api/deprecation-guide/).
+
+## Deprecations and removals for Kubernetes v1.35
+
+### Remove cgroup v1 support
+
+cgroup v2 support has been stable in Kubernetes since v1.25, providing an alternative to the original cgroup v1. While cgroup v1 provided the initial resource control mechanism, it suffered from well-known inconsistencies and limitations. cgroup v2 introduces a unified hierarchy, improved resource isolation, and is the foundation for modern features, making the legacy cgroup v1 support ready for removal. This removal will only impact cluster administrators running nodes on older Linux distributions that do not support cgroup v2; the `kubelet`  will fail to start on these nodes. Administrators must migrate their nodes to systems with cgroup v2 enabled, and more details on compatibility requirements will be available in a detailed blog post soon.
+
+You can find more in [KEP-5573: Remove cgroup v1 support](https://kep.k8s.io/5573)
+
+### Deprecate ipvs mode in kube-proxy
+
+This KEP proposes the deprecation of the IPVS mode in kube-proxy, which has historically provided high-performance service load balancing in Kubernetes. Over time, the maintenance cost, complexity, and limited feature parity with other proxy modes, particularly iptables and the emerging eBPF-based approaches, have made IPVS less practical to support. The deprecation aims to streamline kube-proxy’s codebase, reduce technical debt, and align Kubernetes networking with modern, scalable, and actively maintained technologies that better meet current performance and observability needs.
+
+You can find more in [KEP-5495: Deprecate ipvs mode in kube-proxy](https://kep.k8s.io/5495)
+
+### Fix inconsistent container ready state after kubelet restart
+
+When a kubelet restarts, the expectation is that running Pods continue operating without disruption. However, today a kubelet restart temporarily marks all Pods on that Node as not Started or Ready, even if they are functioning normally. This can trigger unnecessary service interruptions and affect workload stability.
```

Comment:
```suggestion
When a kubelet restarts, it is expected that running Pods continue operating without disruption. However, today, a kubelet restart temporarily marks all Pods on that Node as not Started or Ready, even if they are functioning normally. This can trigger unnecessary service interruptions and affect workload stability.
```